src/app/api/openGraphImage/OpenGraphImage.tsx

```diff
@@ -0,0 +1,51 @@
+interface OpenGraphImageProps {
```

Comment:
I'd like to avoid inline styling, but let's just add a comment saying this is just an example for now i.e mock, and we'll just update later. I'm pretty sure the image will be an illustration and not contain any text or styling.

Reply:
The CSS support is limited for this element (see https://github.com/vercel/satori?tab=readme-ov-file#css), so I am not sure it supports CSS modules.

Also, the whole point of generating the image is that we can use dynamic data from the page (like title and description). Otherwise we could have just used a static image asset.